server/api/scheduler.go

```diff
@@ -213,7 +215,9 @@ func (h *schedulerHandler) addEvictOrGrant(w http.ResponseWriter, input map[stri
 	err := h.AddEvictOrGrant(storeID, name)
 	if err != nil {
 		h.r.JSON(w, http.StatusInternalServerError, err.Error())
+		return
 	}
+	h.r.JSON(w, http.StatusOK, "The scheduler is created.")
```

Comment:
Is the original one useless?

Reply:
Since this method does not return an error, after returning user error information to the user, the outer layer will also return "The scheduler is created."

Reply:
after the program returned an error message to the user in the addEvictOrGrant method, the "addEvictOrGrant" method did not return an error message to the caller CreateScheduler, resulting in CreateScheduler The method continues to return "The scheduler is created" information to the user, and since "grant-leader-scheduler" and "evict-leader-scheduler" share the "addEvictOrGrant" method, the problem also occurs in "evict-leader-scheduler".

Reply:
What I mean is can we remove line 206?

Reply:
Maybe can modify the function `addEvictOrGrant` to return error, and then modify the function `CreateScheduler` to return the error in a uniform location?